lib/game/base_game.dart

```diff
@@ -31,7 +31,7 @@ class BaseGame extends Game with FPSCounter {
   final List<Component> _removeLater = [];
 
   /// Current game viewport size, updated every resize via the [resize] method hook
-  Vector2 size;
+  Vector2 gameSize;
```

Comment:
I think it makes sense on the component that it has gameSize named gameSize, but on the game which is another type of object I don't think we have to follow that convention, that the size field on game is the game size should be clear and just adds redundancy to call game.gameSize imho.

Reply:
On a second thought, I think I agree with @spydon !